.env.prod.example

```diff
@@ -42,6 +43,16 @@ EMAIL_HOST_PASSWORD=emailpassword123
 EMAIL_USE_TLS=true
 EMAIL_USE_SSL=false
 
+# S3 configuration
+# Commented are example values if you use Scaleway instead of AWS
+USE_S3=false
+# AWS_ACCESS_KEY_ID=
+# AWS_SECRET_ACCESS_KEY=
```

Comment:
Are these optional, even for AWS? if not, I think they should be uncommented for clarity